### PR TITLE
Add debilitation effects

### LINKED ISSUE #7628: Please make the effects of Vicious, Tactical and Precise Debilitations (Rogue)

### PR DESCRIPTION
**Pull request by bcronin90**
Add effects to the following feats:
https://2e.aonprd.com/Feats.aspx?ID=578
https://2e.aonprd.com/Feats.aspx?ID=580
https://2e.aonprd.com/Feats.aspx?ID=581

Closes #7628

Limitations:
- When applying the precision damage effect from Precise Debilitations, this does not differentiate between different rogues with this feat applying this effect.

### CODE REVIEW

File: packs/data/feat-effects.db/effect-vicious-debilitations-bludgeoning.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+    "_id": "Rgt9hH3W1oh9dvku",
+    "img": "systems/pf2e/icons/spells/blade-barrier.webp",
+    "name": "Effect: Vicious Debilitations (Bludgeoning)",
```
[the remark this answers is not in the document]
Comment (bcronin90) [reply packs/data/feat-effects.db/effect-vicious-debilitations-bludgeoning.json:4]: Thanks, I'll read up on ChoiceSets, same for the other PR

File: packs/data/feats.db/precise-debilitations.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You carefully aim and gracefully deliver your debilitations. Add the following debilitations to the list you can choose from when you use @UUID[Compendium.pf2e.actionspf2e.Debilitating Strike]{Debilitating Strike}.</p>\n<ul>\n<li><strong>Debilitation</strong> The target takes an additional 2d6 precision damage from your attacks.</li>\n<li><strong>Debilitation</strong> The target becomes @UUID[Compendium.pf2e.conditionitems.Flat-Footed]{Flat-Footed}.</li>\n</ul>"
+            "value": "<p>You carefully aim and gracefully deliver your debilitations. Add the following debilitations to the list you can choose from when you use @UUID[Compendium.pf2e.actionspf2e.Debilitating Strike]{Debilitating Strike}.</p>\n<ul>\n<li><strong>Debilitation</strong> The target takes an additional @UUID[Compendium.pf2e.feat-effects.Effect: Precise Debilitations]{2d6 precision damage} from your attacks.</li>\n<li><strong>Debilitation</strong> The target becomes @UUID[Compendium.pf2e.conditionitems.Flat-Footed]{Flat-Footed}.</li>\n</ul>"
```
[the remark this answers is not in the document]
Comment (bcronin90) [reply packs/data/feats.db/precise-debilitations.json]: Thanks for the hint!

File: packs/data/feat-effects.db/effect-vicious-debilitations.json [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+        "target": null,
+        "tokenIcon": {
+            "show": true
+        },
+        "traits": {
+            "value": []
+        }
+    },
+    "type": "effect"
+}
```
Comment (TikaelSol) [inline packs/data/feat-effects.db/effect-vicious-debilitations.json:60]:
```suggestion
}

```

File: packs/data/feats.db/tactical-debilitations.json [hunk 4]
```diff
@@ -11,7 +11,7 @@
         },
         "category": "class",
         "description": {
-            "value": "<p>You learn new debilitations that grant you tactical advantages against your foes. Add the following debilitations to the list you can choose from when you use @UUID[Compendium.pf2e.actionspf2e.Debilitating Strike]{Debilitating Strike}.</p>\n<ul>\n<li><strong>Debilitation</strong> The target can't use reactions.</li>\n<li><strong>Debilitation</strong> The target can't flank or contribute to allies' flanking.</li>\n</ul>"
+            "value": "<p>You learn new debilitations that grant you tactical advantages against your foes. Add the following debilitations to the list you can choose from when you use @UUID[Compendium.pf2e.actionspf2e.Debilitating Strike]{Debilitating Strike}.</p>\n<ul>\n<li><strong>Debilitation</strong> The target can't use reactions.</li>\n<li><strong>Debilitation</strong> The target @UUID[Compendium.pf2e.feat-effects.Effect: Tactical Debilitations (No Flanking)]{can't flank} or contribute to allies' flanking.</li>\n</ul>"
```
[the remark this answers is not in the document]
Comment (bcronin90) [reply packs/data/feats.db/tactical-debilitations.json]: Missed that one, thanks.

**Pull request merged**